### ISSUE TITLE
doesn't add?

### ISSUE THREAD

**Issue by rdp** [vertiginous/pik#13]
<pre>C:\Documents and Settings\packrd.pik>rm config.yml

C:\Documents and Settings\packrd.pik>pik add
This version has already been added.
## C:\Documents and Settings\packrd.pik>cat config.yml

"186: ruby 1.8.6 (2009-03-31 patchlevel 368) [i386-mingw32]": {}

?
Thanks!

**Comment by vertiginous** [comment]
Fixed in 3349705ac8f28697fb2fc2d2107338fc205fa833

```
C:\>pik implode
Are you sure you want pik to implode? This will remove 'C:/Documents and Settings/gthiesfeld/.pik'. [Yn] y

C:\>pik ls
creating C:\Documents and Settings\gthiesfeld/.pik

C:\>pik add
Adding:  186: ruby 1.8.6 (2009-03-31 patchlevel 368) [i386-mingw32]'
 Located at:  c:/ruby/186-p368-mingw32/bin

C:\>pik add -i
1. e]nter a path
2. s]earch
3. q]uit
2
Enter a search path
C:\ruby\\*\bin
Add 'C:/ruby/185-p012-mswin32/bin'? [Yn] y

Adding:  185: ruby 1.8.5 (2006-12-25 patchlevel 12) [i386-mswin32]'
 Located at:  C:/ruby/185-p012-mswin32/bin
Add 'C:/ruby/186-p287-mswin32/bin'? [Yn] y
Adding:  186: ruby 1.8.6 (2008-08-11 patchlevel 287) [i386-mswin32]'
 Located at:  C:/ruby/186-p287-mswin32/bin
Add 'C:/ruby/186-p368-mingw32/bin'? [Yn] y
This version has already been added.
Add 'C:/ruby/191-p129-mingw32/bin'? [Yn] y
Adding:  191: ruby 1.9.1p129 (2009-05-12 revision 23412) [i386-mingw32]'
 Located at:  C:/ruby/191-p129-mingw32/bin
Add 'C:/ruby/191-p243-mingw32/bin'? [Yn] y
Adding:  191: ruby 1.9.1p243 (2009-07-16 revision 24175) [i386-mingw32]'
 Located at:  C:/ruby/191-p243-mingw32/bin

1. e]nter a path
2. s]earch
3. q]uit
3

Quitting...
```

**Comment by vertiginous** [comment]
Released v 0.1.1 gem.

**Comment by rdp** [comment]
works great now
